Tests/com/drew/metadata/exif/SamsungType2MakernoteTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2002-2019 Drew Noakes and contributors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ *
+ * More information about this project is available at:
+ *
+ *    https://drewnoakes.com/code/exif/
+ *    https://github.com/drewnoakes/metadata-extractor
+ */
+
+package com.drew.metadata.exif;
+
+import com.drew.imaging.jpeg.JpegSegmentData;
+import com.drew.imaging.jpeg.JpegSegmentReader;
+import com.drew.imaging.jpeg.JpegSegmentType;
+import com.drew.lang.StreamReader;
+import com.drew.metadata.Metadata;
+import com.drew.metadata.exif.makernotes.SamsungType2MakernoteDirectory;
+import org.junit.Test;
+
+import java.io.BufferedInputStream;
+import java.io.File;
+import java.io.FileInputStream;
+
+import static org.junit.Assert.assertNotNull;
+
+public class SamsungType2MakernoteTest
```

Comment:
Thanks for including a unit test, but actually we don't generally add images to this repo as the repo becomes too large. We have a separate repo, https://github.com/drewnoakes/metadata-extractor-images, which has lots of images and the output from this library, the output from the .NET version, and the differences between them where such differences exist.

Please remove the test image file and the unit test from here. If you're happy to share the image, I'll add it to the image library which will allow us to track the output over time and verify there is no regression. It will also encourage a fix on the .NET side, to stamp out the difference in outputs.

Reply:
Thank you for your kind explanation.
and I'm glad to share the image!

Reply:
Thanks very much!